charts/spire/charts/spire-server/values.yaml

```diff
@@ -182,6 +186,37 @@ dataStore:
       ## @param dataStore.sql.externalSecret.key The key of the secret object whose value is the dataStore.sql password
       key: ""
 
+    ## @param dataStore.sql.maxOpenConns The maximum number of open db connections
+    maxOpenConns: 100
+    ## @param dataStore.sql.maxIdleConns The maximum number of idle connections in the pool
+    maxIdleConns: 2
+    ## @param dataStore.sql.connMaxLifetime The maximum amount of time a connection may be reused. If 0, time is unlimited
+    connMaxLifetime: 0
+    ## @param dataStore.sql.disableMigration True to disable auto-migration functionality
+    disableMigration: false
+
+    readOnly:
+      ## @param dataStore.sql.readOnly.enabled Set to true to configure a readOnly dartabase connection
+      enabled: false
+      ## @param dataStore.sql.readOnly.host Only used by "postgres" or "mysql"
+      host: ""
+      ## @param dataStore.sql.readOnly.port If 0 (default), it will auto set to 5432 for postgres and 3306 for mysql. Only used by those databases.
+      port: 0
+      ## @param dataStore.sql.readOnly.username Only used by "postgres" or "mysql"
+      username: spire
+      ## @param dataStore.sql.readOnly.password Only used by "postgres" or "mysql"
+      password: ""
+      ## @param dataStore.sql.readOnly.options [array] Only used by "postgres" or "mysql"
+      options: []
+      ## When an external source creates the secret. The secret should reside in the same namespace as the spire server
+      externalSecret:
+        ## @param dataStore.sql.readOnly.externalSecret.enabled Enable external secret for datastore creds
+        enabled: false
+        ## @param dataStore.sql.readOnly.externalSecret.name The name of the secret object
+        name: ""
+        ## @param dataStore.sql.readOnly.externalSecret.key The key of the secret object whose value is the dataStore.sql password
+        key: ""
```

Comment:
Does the read only need a separate externalSecret option?

Reply:
The ro password could be in a separate secret. It can be combined too though if you use the same name and different keys.

Reply:
I double checked.... The bitnami postgresql chart lets you create one user via values and loads the pw into its secret. If a user wants to configure another ro user, then they would have to do so manually and load it into a different secret. So, the functionality is required in at least one configuration. So, I think we want these options.